[SEC-31445] severity modifiers doc (#37170)
DataDog/documentation: content/en/security/automation_pipelines/_index.md

Reviewer: ```suggestion
- link: "/security/automation_pipelines/mute"
tag: "Documentation"
text: "Mute Rules"
- link: "/security/automation_pipelines/security_inbox"
tag: "Documentation"
text: "Add to Security Inbox Rules"
- link: "/security/automation_pipelines/set_due_date"
tag: "Documentation"
text: "Set Due Date Rules"
- link: "/security/automation_pipelines/create_ticket"
tag: "Documentation"
text: "Ticket Creation Rules"
- link: "/security/automation_pipelines/modify_severity"
tag: "Documentation"
text: "Severity Modifier Rules"
```
I noticed that these links are otherwise in the same order that they appear in the sidebar, so I thought I would apply that here too for parallelism's sake. We could also consider moving the position in the sidebar, if you'd rather this new page appear higher in all these other places, of course; I just care about them matching so these links are easier to navigate.

Author: Thanks for catching the ordering inconsistencies. I've reordered these, but kept Severity Modifiers first rather than moving it to match the old sidebar order.

Reason: these automation rules types are executed in order (severity modifiers -> mute -> due date -> inbox -> ticketing), and the UI already lists them in that execution order.

So I reordered the sidebar/menu instead to match the pipeline order, rather than reordering `_index.md` to match a sidebar order that was really just chronological-by-ship-date.

Reviewer: ```suggestion
- **Reduce time to remediation**: Get findings into engineering queues immediately after detection, eliminating the delay between discovery and assignment.

### Adjust finding severities to reflect your business context

Override the default severity of findings to match your organization's risk profile. This allows you to:

- **Downgrade low-risk findings**: Reduce the severity of findings on isolated environments, which pose limited real-world risk.
- **Upgrade high-value targets**: Increase the severity of findings on critical systems, such as databases containing personally identifiable information or services with elevated compliance requirements.
- **Calibrate severity to your organization's priorities**: Establish consistent severity standards to reflect your organization's priorities, rather than relying solely on out-of-the-box scoring.

## Further reading
```

## Use cases

### Adjust finding severities to reflect your business context

Override the default severity of findings to match your organization's risk profile. This allows you to:

> - **Downgrade low-risk findings**: Reduce the severity of findings on isolated environments which pose limited real-world risk.

Reviewer: ```suggestion
- **Downgrade low-risk findings**: Reduce the severity of findings on isolated environments, which pose limited real-world risk.
```

> - **Upgrade high-value targets**: Increase the severity of findings on critical systems, such as databases containing PII or services with elevated compliance requirements.

Reviewer: ```suggestion
- **Upgrade high-value targets**: Increase the severity of findings on critical systems, such as databases containing personally identifiable information or services with elevated compliance requirements.
```

> - **Calibrate severity to your organization's priorities**: Establish consistent severity standards which reflect your organization's priorities rather than relying solely on out-of-the-box scoring.

Reviewer: ```suggestion
- **Calibrate severity to your organization's priorities**: Establish consistent severity standards to reflect your organization's priorities, rather than relying solely on out-of-the-box scoring.
```

## Use cases

> ### Adjust finding severities to reflect your business context
> 
> Override the default severity of findings to match your organization's risk profile. This allows you to:
> 
> - **Downgrade low-risk findings**: Reduce the severity of findings on isolated environments, which pose limited real-world risk.
> - **Upgrade high-value targets**: Increase the severity of findings on critical systems, such as databases containing personally identifiable information or services with elevated compliance requirements.
> - **Calibrate severity to your organization's priorities**: Establish consistent severity standards to reflect your organization's priorities, rather than relying solely on out-of-the-box scoring.
> 

Reviewer: ```suggestion

```
Moving this to the end of the page again for the same reason
